app/main.py

```diff
@@ -31,16 +33,38 @@ def knowledge_panel(
         panels = []
         if facet_tag in HungerGameFilter.list():
             panels.append(
-                hunger_game_kp(hunger_game_filter=facet_tag, value=value_tag, country=country)
+                hunger_game_kp(
+                    hunger_game_filter=facet_tag,
+                    value=value_tag,
+                    sec_facet=sec_facet_tag,
+                    sec_value=sec_value_tag,
```

Comment:
It would be better to stick to same parameters name for every function. (see polymorphism, that is having same interface, enables nice patterns)

So I would rename hunger_game_filter and value to facet and value.

Guess what ? Then you could have a loop here ;-)